### PR TITLE
Prepare to move to Adobe org.

### PR DESCRIPTION
**Pull request by dompuiu**

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -13,8 +13,7 @@ if (process.env.TRAVIS) {
     'SL_EDGE',
     'SL_CHROME',
     'SL_FIREFOX',
-    'SL_ANDROID',
-    'SL_SAFARI'
```
Comment (Aaronius) [inline karma.conf.js]: y u no safari?

Comment (dompuiu) [reply karma.conf.js]: If you look in the master, the CI is failing.

Comment (Aaronius) [reply karma.conf.js]: y u no fix?

Comment (dompuiu) [reply karma.conf.js]: I've locked down the MacOS version. This means when newer Safari versions will appear, we will still test on Safari 12.

**Pull request merged**